qiita_pet/templates/sitebase.html

```diff
@@ -239,7 +239,8 @@
       {% block content %}{% end %}
     </div>
 
-    <div id="footer">
+    <div id="footer" style="font-size: 12px; text-align: center; z-index: 1000;">
+      Thank you for using Qiita. For citations point to <a href="http://qiita.microbio.me">http://qiita.microbio.me</a>.
```

Comment:
https: None except that we do not have it running there. If we decide to do that we can always redirect from http -> https so http at this point is safer.